compiler/lib-wasm/generate.ml

```diff
@@ -1195,14 +1194,7 @@ module Generate (Target : Target_sig.S) = struct
     List.rev_append context.other_fields (imports @ constant_data)
 end
 
-let init () =
-  let l =
-    [ "caml_callback", "caml_trampoline"
-    ; "caml_make_array", "caml_array_of_uniform_array"
-    ]
-  in
-  Primitive.register "caml_array_of_uniform_array" `Mutable None None;
```

Comment:
We no longer specify anywhere that `caml_array_of_uniform_array` / `caml_make_array` is `Mutable`.

Reply:
reintroduced